src/tonic/mod.rs

```diff
@@ -38,7 +37,7 @@ impl Qdrant for QdrantService {
     }
 }
 
-pub fn init(toc: Arc<TableOfContent>, settings: Settings) -> std::io::Result<()> {
+pub fn init(toc: Arc<TableOfContent>, host: String, grpc_port: u16) -> std::io::Result<()> {
```

Comment:
just stylish propose about naming. `grpc_host&grpc_port` or `host&port`

Reply:
sorry, I see that host is used for rest api too. Maybe use `rest_port` instead `port` in the whole qdrant?